addons/mail/static/src/discuss/core/common/discuss_channel_model.js

```diff
@@ -82,6 +82,9 @@ export class DiscussChannel extends Record {
     get membersThatCanSeen() {
         return this.channel_member_ids;
     }
+    get isChatChannel() {
+        return ["chat", "group"].includes(this.channel?.channel_type);
+    }
```

Comment:
Unrelated to this PR, but we're trying to reduce the differences between group and channel, so whatever we do with `isChatChannel` doesn't play well with that.